src/index.html

```diff
@@ -5,9 +5,329 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+
+    <header class="header">
+      <div class="header__logo-with-icons">
+        <img
+          src="./images/met-logo.png"
+          alt="logo"
+          class="logo"
+        >
+
+        <div class="icons">
+          <img
+            src="./images/phone.svg"
+            alt="logo"
+            class="phone"
+          >
+          <a href="#menu">
+            <img
+              src="./images/burger.svg"
+              alt="logo"
+              class="burger"
+            >
+          </a>
+        </div>
+      </div>
+
+      <h1 class="header__title">Welcome to The MET</h1>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="top-row">
+        <img
+          src="./images/met-logo.png"
+          alt="logo"
+          class="menu-logo"
+        >
+
+        <a href="#">
+          <img
+            src="./images/Icon-Close.svg"
+            alt="logo"
+            class="menu-close"
+          >
+        </a>
+      </div>
+
+      <ul class="menu-list">
+        <a href="#" class="item-link">
+          <li class="menu-item">Home</li>
+        </a>
+        <a href="#" class="item-link">
+          <li class="menu-item">About Us</li>
+        </a>
+        <a href="#" class="item-link">
+          <li class="menu-item">Now on view</li>
+        </a>
+        <a href="#" class="item-link">
+          <li class="menu-item">Explore Online</li>
+        </a>
+        <a href="#" class="item-link">
+          <li class="menu-item">Contacts</li>
+        </a>
+      </ul>
+
+      <div class="menu-info">
+        <p class="info-item">The Met Fifth Avenue<br>+1 212-535-7710</p>
+
+        <p class="info-item">The Met Cloisters<br>+1 212-923-3700</p>
+
+        <a href="#" class="info-link">call to order</a>
+      </div>
+    </nav>
+
+    <main class="main">
+
+      <section class="schedule">
+
+        <div class="schedule__block-left">
+          <span class="span">Plan your visit</span>
+          <h2 class="h2">Museum hours</h2>
+        </div>
+
+        <div class="schedule__block-right">
+          <h3 class="h3">The Met Fifth Avenue</h3>
+
+          <div class="first">
+            <p class="p4"><strong class="strong">10 am - 5 pm</strong> Sun, Mon, Tue, Thu</p>
+            <p class="p4"><strong class="strong">10 am - 9 pm</strong> Fri, Sat</p>
+            <p class="p4"><strong class="strong">Closed </strong>Wen</p>
+          </div>
+
+          <div class="second">
```

Comment:
`first, second` - bad class name.
also can't find any link between element from 109 line and 88! But that h3 is an element of whole block `schedule `